Assignment6Test.java

```diff
@@ -0,0 +1,56 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
```

Comment:
Maybe we should add some more test cases to make sure we cover all the code. Maybe we should also add some longer examples.